spec/package-manager-spec.coffee

```diff
@@ -357,6 +357,26 @@ describe "PackageManager", ->
             expect(atom.keymaps.findKeyBindings(keystrokes:'ctrl-n', target:element1[0])[0].command).toBe 'keymap-2'
             expect(atom.keymaps.findKeyBindings(keystrokes:'ctrl-y', target:element3[0])).toHaveLength 0
 
+      describe "when the package is listed in the config 'core.packagesWithKeymapsDisabled'", ->
+        it "doesn't load the keymaps", ->
+          atom.config.set("config.packagesWithKeymaps", ["package-with-keymaps"]);
```

Comment:
Should be `atom.config.set("config.packagesWithKeymapsDisabled", ["package-with-keymaps"]);`